lib/hanami/cli/commands/app/db/utils/postgres.rb

```diff
@@ -44,7 +44,15 @@ def exec_dump_command
                 system_call.call(
                   "pg_dump --schema-only --no-privileges --no-owner --file #{structure_file} #{escaped_name}",
                   env: cli_env_vars
-                )
+                ).tap do
+                  next unless it.successful?
```

Comment:
Be mindful of what language features you use in library code. This is unsupported by 3/4 of the allowed versions in the gemspec.